docs/get-started/deployment-and-usage.md

```diff
@@ -244,7 +244,7 @@ the default interval is set to 60s which is also the default when no
 `-sleep-interval` is specified. Also, the configuration file is re-read on
 each iteration providing a simple mechanism of run-time reconfiguration.
 
-### TLS authentication
+### Tls authentication
```

Comment:
Communication security with TLS ?

Reply:
OK, changed. Even if this patch is supposed to be just about capitalization 😄